stylesheets/pages/modal.css

```diff
@@ -0,0 +1,23 @@
+.modal-wrapper {
+  display: none;
+  justify-content: center;
+  align-items: center;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: var(--black-light);
+}
+
+.flex {
+  display: flex;
+}

```

Comment:
This class would be better suited in utils.css rather than on component styling.

Reply:
Due to the specificity, I have used it here.